dockstore-integration-testing/src/test/java/io/dockstore/webservice/WebhookIT.java

```diff
@@ -97,10 +97,14 @@ public void testWorkflowMigration() throws Exception {
         // Refresh should work
         workflow = workflowApi.refresh(workflow.getId());
         assertEquals("Workflow should be FULL mode", Workflow.ModeEnum.FULL, workflow.getMode());
-        assertTrue("All versions should be legacy", workflow.getWorkflowVersions().stream().allMatch(workflowVersion -> workflowVersion.isLegacyVersion()));
+        assertTrue("All versions should be legacy", workflow.getWorkflowVersions().stream().allMatch(WorkflowVersion::isLegacyVersion));
+
+        // seems like handleGitHubRelease has issue with versions that already exist
+        testingPostgres.runUpdateStatement("delete from workflowversion where parentid = " + workflow.getId());
```

Comment:
FYI @agduncan94  looks like handleGitHubRelease might be creating duplicates

Reply:
Andrew and I were looking at this and I'm trying to fix.